Sources/PointFree/Account/Account.swift

```diff
@@ -906,6 +907,64 @@ private func inviteRowView(_ invite: TeamInvite) -> Node {
     ])
 }
 
+private func addTeammateToSubscriptionRow(_ data: AccountData) -> [Node] {
+  guard !data.subscriberState.isEnterpriseSubscriber else { return [] }
+  guard let subscription = data.stripeSubscription else { return [] }
+  let invitesRemaining = subscription.quantity - data.teamInvites.count - data.teammates.count
+  guard invitesRemaining == 0 else { return [] }
+  guard let amount = subscription.plan.tiers.min(by: { $0.amount < $1.amount })?.amount else { return [] }
+
+  let interval = subscription.plan.interval == .some(.year) ? "year" : "month"
+  let amountPerPeriod = "$\(amount.rawValue / 100) per \(interval)"
```

Comment:
can you double check how I am computing the amount and interval in the above 3 lines of code?

Reply:
don’t you have to account for if the current quantity is 1 it will be $14 for the additional seat, otherwise $16? so the difference between the max and min